daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Command.scala

```diff
@@ -49,13 +48,13 @@ private[lf] object Command {
   /** Fetch a template, not by interface */
   final case class FetchTemplate(
       templateId: Identifier,
-      coid: SContractId,
+      coid: ContractId,
   ) extends Command
 
   /** Fetch a template, by interface */
   final case class FetchInterface(
       interfaceId: Identifier,
-      coid: SContractId,
+      coid: ContractId,
```

Comment:
It would have been nice to split cleanups like this into a separate PR. They cause a lot of noise, hiding the substantive changes, and unless I am misunderstanding something, are not essential to the upgrading feature.

Reply:
I will revert that.

Reply:
11097500c4fca0916d50b8f69b76f8a2c4a36cf7